src/ucontext.rs

```diff
@@ -22,4 +23,13 @@ impl UContext {
         };
         Errno::result(res).map(drop)
     }
+
+    #[allow(mutable_transmutes)]
```

Comment:
Thanks you. I have missed that in some other code of mine as well. Any other comments?